build.sbt

```diff
@@ -1112,12 +1112,14 @@ lazy val `interpreter-dsl` = (project in file("lib/scala/interpreter-dsl"))
 
 val truffleRunOptions = if (java.lang.Boolean.getBoolean("bench.compileOnly")) {
   Seq(
+    "-ea",
```

Comment:
Will this option be appended also inside the tests of `language-server`? `truffleRunOptions` seems to be used just by all the `runtime-*` projects, and `engine-runner`, but not by, e.g., `language-server`.

Reply:
Enabled in 89a2c3b797ea